server/graphql/types/src/types/item_variant.rs

```diff
@@ -0,0 +1,121 @@
+use async_graphql::*;
+use repository::item_variant::{
+    item_variant_row::ItemVariantRow, packaging_variant_row::PackagingVariantRow,
+};
+pub struct PackagingVariantNode {
+    pub packaging_variant: PackagingVariantRow,
+}
+
+pub struct ItemVariantNode {
+    pub item_variant: ItemVariantRow,
+}
+
+#[Object]
+impl ItemVariantNode {
+    pub async fn id(&self) -> &String {
+        &self.item_variant.id
+    }
+
+    pub async fn item_id(&self) -> &String {
+        &self.item_variant.item_link_id // TODO join to item for item_id
```

Comment:
Just do an item loader? Not sure what we'll actually use this for?

I think I remember @andreievg saying that repository or service layer should map the item_link_id to item_id.
Is there a wiki about this?

Not for this PR but I realise I'm not clear on how we are doing it :)

Reply:
I think my point was if we're generally loading the item_variant from a loader on item, looking up or even providing item_id on the node feels a bit redundant. I guess, we just wait to see what we actually need for the front end. Maybe we can even just get rid of this for now until we need it.

Definitely keen for that Loader write up. `link_id` one would be good too!